test/uk/gov/hmrc/traderservices/controllers/ImportQuestionsFormSpec.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.traderservices.controllers
+
+import play.api.data.FormError
+import uk.gov.hmrc.play.test.UnitSpec
+import uk.gov.hmrc.traderservices.models.{ImportFreightType, ImportPriorityGoods, ImportRequestType, ImportRouteType}
+import uk.gov.hmrc.traderservices.support.FormMatchers
+import play.api.data.Form
+
+class ImportQuestionsFormSpec extends UnitSpec with FormMatchers {
+
+  def validate[A](form: Form[A], formInput: Map[String, String], formOutput: A): Unit = {
+    form.bind(formInput).value shouldBe Some(formOutput)
+    form.fill(formOutput).data shouldBe formInput
+  }
+
+  def validate[A](form: Form[A], fieldName: String, formInput: Map[String, String], errors: Seq[String]): Unit = {
+    form.bind(formInput).value shouldBe None
+    form.bind(formInput).errors should haveOnlyErrors(errors.map(e => FormError(fieldName, e)): _*)
+  }
+
+  "Import questions forms" should {
+
+    "bind some requestType and return ImportRequestType and fill it back" in {
```

Comment:
It's not on the page but just from cmd clicking to check these I noticed the ALVS option is still here on this page - think it can be safely removed 
<img width="739" alt="image" src="https://user-images.githubusercontent.com/55190161/95446482-45567c80-0958-11eb-8974-20973b105095.png">

Reply:
<img width="616" alt="image" src="https://user-images.githubusercontent.com/55190161/95446525-5606f280-0958-11eb-9b3f-680bb69b2aa5.png">